### PR TITLE
Chore/remove legacy material

### PR DESCRIPTION
**Pull request by mimse**

### CODE REVIEW

File: libs/watt/src/lib/components/button/watt-button.component.scss [hunk 1]
```diff
@@ -54,47 +54,8 @@
...
-    }
-
-    .content-wrapper--loading {
-      visibility: hidden;
-    }
-
-    watt-icon {
-      margin: -5px; // to offset the icons inherent padding of 5px
-    }
-  }
```
Comment (dzhavat) [inline libs/watt/src/lib/components/button/watt-button.component.scss:94]: issue: the primary button changes to sharp corners on hover

image

Comment (dzhavat) [inline libs/watt/src/lib/components/button/watt-button.component.scss:94]: issue: clicking the primary button doesn't trigger the ripple effect. It does when clicking the secondary button.

(the gif bellow shows primary button in this PR, followed by primary button in deployed Storybook)

issue

File: libs/watt/src/lib/components/button/watt-button.component.scss [hunk 2]
```diff
@@ -54,47 +54,8 @@
...
```
Comment (dzhavat) [inline libs/watt/src/lib/components/button/watt-button.component.scss:57]: issue: button outline is missing (when a button is focused with Tab) on all button types (primary, secondary, text, icon)

File: libs/watt/src/lib/components/button/watt-button.component.ts [hunk 3]
```diff
@@ -38,9 +38,9 @@ export type WattButtonType = 'button' | 'reset' | 'submit';
   template: `
     <button
       mat-button
-      class="watt-button--{{ variant }}"
       [disabled]="disabled"
       [type]="type"
+      [color]="variant"
```
[the remark this answers is not in the document]
Comment (mimse) [reply libs/watt/src/lib/components/button/watt-button.component.ts:43]: It does not work if I do not provide the color

**Review by dzhavat — APPROVED** [review]

**Pull request merged**